src/foraging_gui/Foraging.py

```diff
@@ -4613,7 +4613,10 @@ def _generate_upload_manifest(self, session: Session):
             if not hasattr(self, 'project_name'):
                 self.project_name = 'Behavior Platform'
 
-            schedule = self.behavior_session_model.date.strftime("%Y-%m-%d_%H-%M-%S").split('_')[0]+'_20-30-00'
+            # Upload time is 8:30 tonight, plus a random offset over a 30 minute period
```

Comment:
might be useful to add a why in the comment here for future developers

Reply:
done!